### DISCUSSION TITLE
Can not install Hacmini to selected U disk

### DISCUSSION THREAD

**Discussion by xugongkai1998** [osy86/HaC-Mini#1048] [Q&A]
**Describe the bug**
I made the installation media, but I can't install Hacmini to the U disk （with default  installation options）

**Expected behavior**
install Hacmini to U disk

**Environment:**
 - Running OSX version:  MacOs BigSur 11.6(20g165)
 - Target drive OSX version: MacOS_Ventura_13.0_22A380
 - HaC Mini installer version: v3.2.2

**Additional context**
I followed the wizard here to perform the installation：
<img width="1137" alt="image" src="https://user-images.githubusercontent.com/96639799/201716821-f6640931-cdb3-4a35-af1c-9b2e18a5bece.png">
When I followed the wizard to this step, something unexpected happened：
<img width="622" alt="image" src="https://user-images.githubusercontent.com/96639799/201717379-13ed497e-31ef-4d3e-a7f3-8a7c9ab86f9e.png">

In addition, Ventura's installer is not downloaded directly from the AppStore, but the downloaded iso image. I mounted it and dragged it into the Application folder, and in the terminal interface, when creating the installation medium, I did not any exception occurs.

I tried changing the USB stick and recreating the installation media, but still the same error. I don't have a solution right now. So wanted to see if you could help with this issue :)
**Logs**
here is log file：
[log.log](https://github.com/osy/HaC-Mini/files/10005247/log.log)

**Comment by osy** [comment]
Please try an older version of macOS and update to Ventura.

**Reply by KrisCris** [reply]
I am actually not sure how to download the old installer. System Update keeps saying something wrong, as a result, I can't complete the download...

**Comment by Medizinmann-MD** [comment]
As I understand you running Big Sur?
Did you upgrade your system with HacMini 3.2.2?
Than you could just use Upgrade you existing install.

To create a new installer you could use GibMacOS to load older Version i.e. Big Sur...
https://intozoom.com/gibmacos-create-bootable-usb/
...and upgrade that after install.
